lib/ui/compositing.dart

```diff
@@ -21,6 +21,20 @@ class Scene extends NativeFieldWrapperClass1 {
   @pragma('vm:entry-point')
   Scene._();
 
+  /// Creates a GPU resident image from this scene.
+  ///
+  /// {@macro dart.ui.painting.Picture.toGpuImage}
+  Image toGpuImage(int width, int height) {
```

Comment:
I am not very sure how shall we use this new API to reuse expensive drawings across frames. Is there an example? Thanks!

Reply:
No, that would probably come at a higher level in the framework or in a package.

Reply:
Sure. I can contribute once I understand it.